framework/src/iotbus/iotbus_gpio.c

```diff
@@ -155,6 +156,58 @@ int iotbus_gpio_set_direction(iotbus_gpio_context_h dev, iotbus_gpio_direction_e
 	return IOTBUS_ERROR_NONE;
 }
 
+/**
+ * @brief Registers signal for current process based on rising or falling edge on the gpio line
+ */
+#ifndef CONFIG_DISABLE_SIGNALS
+int iotbus_gpio_register_signal(iotbus_gpio_context_h dev, iotbus_gpio_edge_e edge)
+{
+        int ret = -1;
```

Comment:
Here are spaces. Let's change them to a tab.

Reply:
Fixed